rules/prefer-ramda-boolean.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const R = require('ramda');
+const isBooleanLiteral = require('../ast-helper').isBooleanLiteral;
+
+const report = (instead, prefer) => `Instead of \`() => ${instead}\`, prefer \`${prefer}\``;
+const getAlternative = R.applyTo(R.__, R.compose(R.toUpper, R.head, R.toString));
+
+// :: Node -> Boolean
+const onlyReturnsBoolean = R.where({
+    type: R.equals('BlockStatement'),
+    body: R.both(
+        R.propEq('length', 1),
+        R.where({
+            0: R.where({
+                type: R.equals('ReturnStatement'),
+                argument: R.both(
+                    R.complement(R.isNil),
+                    isBooleanLiteral
+                )
+            })
+        })
+    )
+});
+
+// Node -> String
+const getRawReturn = R.ifElse(
+    R.propEq('type', 'BlockStatement'),
+    R.path(['body', 0, 'argument', 'value']),
+    R.prop('value')
+);
+
+const create = context => ({
+    ArrowFunctionExpression(node) {
+        const match = R.either(isBooleanLiteral, onlyReturnsBoolean);
+
+        if (match(node.body)) {
+            const instead = getRawReturn(node.body);
+            context.report({
+                node,
+                message: report(instead, getAlternative(instead))
+            });
+        }
+    },
+
+    FunctionExpression(node) {
+        if (onlyReturnsBoolean(node.body)) {
+            const instead = node.body.body[0].argument.value;
+            context.report({
+                node,
+                message: report(instead, getAlternative(instead))
+            });
+        }
+    }
```

Comment:
Is there a reason not to include `FunctionDeclaration`?

Reply:
:white_check_mark: Yes. `FunctionDeclaration` is not a `FunctionExpression`, so, it has no "return". `function(){}` is a declaration and invalid as an expression, for example, but `(function(){})` is a valid expression and the name of the nodes are different. A `FunctionDeclaration` is a statement, so we could't replace it by reference directly.

Reply:
You are right, makes totally sense.